### PR TITLE
add label hover is white as its an action.

### PR DESCRIPTION
**Pull request by akash1810**
looks nicer...

add-label-hover

### CODE REVIEW

File: kahuna/public/js/components/gr-add-label/gr-add-label.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<button class="button-shy"
-        ng:class="{'small': ctrl.grSmall}"
+<button ng:class="{'small': ctrl.grSmall}"
```
Comment (jamesgorrie) [inline kahuna/public/js/components/gr-add-label/gr-add-label.html]: Could we remove this as we don't have a `.small` class.

**Comment by jamesgorrie** [comment]
:+1:

File: kahuna/public/js/components/gr-add-label/gr-add-label.html (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/components/gr-add-label/gr-add-label.html]: This [component](https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-add-label/gr-add-label.js#L47) does. Its possible misleading as small in this case means no button text, just the icon...

**Comment by akash1810** [comment]
merging - revising nomenclature for another PR.

**Pull request merged**